src/Analyzer.Types.ConfigurationTests/Tests/ExcludeSeverityThree.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "Template": "AppServices",
+    "Configuration": "ExcludeSeverityThree",
+    "ReportedFailures": [
+      {
+        "Id": "TA-000001",
+        "Severity": 2
```

Comment:
The idea of specifying the severity here is to validate that configurations that change the severity of rules work properly? I don't see a test case that validates that feature

Reply:
Partially. For the moment it is for either filtering by Id and/or by Severity (which I tested). In the future when re-mapping is supported, that will be used. It also seemed cleaner to keep the values together rather than create a separate object/list, etc.

Reply:
To me, it sounds a bit redundant and confusing to add the severity there, especially if that feature is not yet supported. Because when the tests are executed you should already have access to the rule information, including the severity, to validate the filtering. Imagine also if we change the severity of a rule, we would need to change every single instance in every test case that mentions it

Reply:
@anaismiller thoughts on this?

Reply:
@anaismiller @JohnathonMohr @yane3628 thoughts on this?

Reply:
@VeraBE, @anaismiller and I discussed offline.  I don't think these tests are here anymore, so I think your concern is addressed.